e2e/terraform/modules/e2e/main.tf

```diff
@@ -90,9 +84,21 @@ resource "google_compute_instance" "e2e_instances" {
       agent       = false
     }
     inline = [
-      "cd /home/ubuntu/test-infra/e2e/provision/",
-      "chmod +x init.sh",
-      "sudo -E NEPHIO_REPO_DIR=/home/ubuntu/test-infra NEPHIO_DEBUG=true NEPHIO_RUN_E2E=true ./init.sh"
+      "if [[ $(hostnamectl | grep Fedora) == *Fedora* ]]; then sudo dnf update -y && sudo shutdown -r +1; fi"
```

Comment:
Is there a reason to do this here? Can we include it on the `init.sh` bash script?

Reply:
It is critical that the fedora34 kernel get updated to the latest so that gtp5g module gets installed. I tried doing this in init.sh and in ansible scripts, however, terraform seems to error out indicating that a return code was not returned.